inference-contract/README.md

```diff
@@ -0,0 +1,38 @@
+<!---
+Licensed under Creative Commons Attribution 4.0 International License
+https://creativecommons.org/licenses/by/4.0/
+--->
+
+# NFT and openvino inference contracts using PDO #
+This directory contains the implementation of a Private Data Objects contract family that implements a machine learning (ML) model based digital asset for performing policy-based inferencing operations using the underlying ML model. The protocol used for binding the asset to its use-policy is largely similar to the one used within the [Digital Asset Contract Family](../digital-asset-contract/README.md). The Token Object PDO contract specifies and checks the policies to be followed to use the ML model for the inferecing operation. The asset guardian service is implemented as a web service containing a frontend and backend portions. In this implementation, the ML Model is assumed to be an [OpenVINO based IR](https://docs.openvino.ai/2023.2/openvino_ir.html) model.  The backend of the guardian service in this case implements the OpenVINO model server capable of performing inferencing operations using the OpenVINO IR model. The frontend of the guardian service at a high level performs two functions: 1. Processes the capability package generated by the token object and generates the API for the backend. 2. Implements any input preprocessing and output postprocessing (collectively referred to as model scoring)
```

Comment:
Stylistic: Please wrap these lines at 80 chars. This will make it much easier to review sentence by sentence.